docs/_tutorials/local_development.md

```diff
@@ -71,40 +59,15 @@ Once you've installed a development proxy, run it to begin forwarding requests t
 
-## Listening to workspace events
+### Listening to workspace events
 If you want your app to listen to events like when a reaction is added, when a user mentions your app, or [another Events API event](https://api.slack.com/events), you'll need to enable events for your app.
 
-> The `@slack/events-api` is used for the Events API.
-
-Start by clicking **Event Subscriptions** on the left sidebar. After you toggle the switch, you'll see you need to add a Request URL. This is similar to [adding a Request URL in the previous section](#what-is-a-Request-URL). There are two differences though:
-1. The default path is `/slack/events` instead of `/slack/actions`
-2. The Request URL in this section will need to verify that your endpoint can handle events.
-
-To do this, we've included a command line tool in the `@slack/events-api` package that helps you verify your endpoint.
-
-### Verifying your local endpoint
-If you're using the development proxy as [described above](#using-a-local-request-URL-for-development), you can run the tool from inside your project directory (after the `@slack/events-api` package is installed). You should **only use the command line tool in development**. If your app is up and running, the adapter will automatically respond to challenges.
-
-You can run the command line tool from inside your project directory with the following command:
-
-```
-./node_modules/.bin/slack-verify --secret <signing_secret> [--path=/slack/events] [--port=3000]
-```
-
-You'll need to substitute your own signing secret for `<signing_secret>`. This is found on the **Basic Information** page in the [Create an App section](#create-an-app).
-
-![Signing Secret](signing-secret.png "Signing Secret on Basic Information page")
-
-The path and port values are optional. If your request URL includes a different path, you should specify it with `--path=/my/path/here` (no brackets). Similarly, if your development proxy is forwarding requests to a different port, you should specify it with `--port=8888` (no brackets). If you're using the defaults, you can ignore everything after `<signing_secret>`.
-
-Now that you've verified your endpoint, you can add events.
+Start by clicking **Event Subscriptions** on the left sidebar. If your app is using HTTP to communicate with Slack, after you toggle the switch, you'll need to add a Request URL. This is similar to [adding a Request URL in the **Interactive Components** section](#what-is-a-Request-URL), but as described on this page, Slack will issue a `challenge` request to verify that your app's request URL can respond appropriately to incoming requests. Read through our [Request URL Configuration &amp; Verification](https://api.slack.com/apis/connections/events-api#the-events-api__subscribing-to-event-types__events-api-request-urls__request-url-configuration--verification) documentation for details on how to properly handle a `challenge` request.
```

Comment:
If anyone else if curious, the `&amp;` in `Request URL Configuration &amp; Verification` renders correctly with Jekyll:
<img width="801" alt="image" src="https://user-images.githubusercontent.com/21328/164775336-e9d01c16-a08d-4e88-bd20-f2a8a2e8eaa6.png">